relay-config/src/config.rs

```diff
@@ -2177,15 +2166,10 @@ impl Config {
         self.values.spool.envelopes.max_memory_size.as_bytes()
     }
 
-    /// Number of envelopes to read from disk at once.
-    pub fn spool_envelopes_read_batch_size(&self) -> usize {
-        self.values.spool.envelopes.read_batch_size
-    }
-
     /// Number of encoded envelope bytes that need to be accumulated before
     /// flushing one batch to disk.
-    pub fn spool_envelopes_write_batch_bytes(&self) -> usize {
-        self.values.spool.envelopes.write_batch_bytes.as_bytes()
+    pub fn batch_size_bytes(&self) -> usize {
```

Comment:
Should we keep the naming consistent with the `spool_envelopes` prefix? I don't have strong opinions but just to keep it consistent.

Reply:
Absolutely, it's a method on the top-level `Config` so it should have the prefix 👍.